src/List.ts

```diff
@@ -1,4 +1,4 @@
-import { forOf, Iterable, IterableIterator, ShimIterator } from '@dojo/shim/iterator';
+import {Iterable, IterableIterator, ShimIterator} from '@dojo/shim/iterator';
```

Comment:
minor nit... spaces between the braces

Reply:
**Ugh** WebStorm. I had to reinstall it and it looks like my special dojo2 style settings got disappeared. I'll have to go through the style guide and create a new one 😕